<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>&amp;Title*</string>
         </property>
         <property name="buddy">
          <cstring>titleLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="titleLineEdit">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>&amp;Name*</string>
         </property>
         <property name="buddy">
          <cstring>nameLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="nameLineEdit">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="authorLabel">
         <property name="text">
          <string>&amp;Author*</string>
         </property>
         <property name="buddy">
          <cstring>authorLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="authorLineEdit">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="contactLabel">
         <property name="text">
          <string>&amp;Contact</string>
         </property>
         <property name="buddy">
          <cstring>contactLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="contactLineEdit">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="homepageLabel">
         <property name="text">
          <string>&amp;Homepage</string>
         </property>
         <property name="buddy">
          <cstring>homepageLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="homepageLineEdit">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>&amp;Description</string>
         </property>
         <property name="buddy">
          <cstring>descriptionTextEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPlainTextEdit" name="descriptionTextEdit"/>
       </item>
       <item row="8" column="0">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="versionLabel">
         <property name="text">
          <string>&amp;Version*</string>
         </property>
         <property name="buddy">
          <cstring>v0Box</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="v0Box">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="vd0Label">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="v1Box">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="vd1Label">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="v2Box">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="factorioVersionLabel">
         <property name="text">
          <string>&amp;Factorio Version*</string>
         </property>
         <property name="buddy">
          <cstring>fv0Box</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="fv0Box">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fvdLabel">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fv1Box">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="availabilityLabel">
         <property name="text">
          <string>Mod name is available</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>* means a mandatory field</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>titleLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>authorLineEdit</tabstop>
  <tabstop>contactLineEdit</tabstop>
  <tabstop>homepageLineEdit</tabstop>
  <tabstop>descriptionTextEdit</tabstop>
  <tabstop>v0Box</tabstop>
  <tabstop>v1Box</tabstop>
  <tabstop>v2Box</tabstop>
  <tabstop>fv0Box</tabstop>
  <tabstop>fv1Box</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
